bayesian/utility/db_gateway.py

```diff
@@ -45,9 +45,19 @@ class GraphAnalyses:
             .as('version').in('has_version').dedup().as('package').select('version')
             .coalesce(out('has_snyk_cve').as('cve').select('package','version','cve')
             .by(valueMap()),select('package','version').by(valueMap()));
-            """
+            """,
     }
 
+    def __init__(self):
+        """Initialise GraphAnalyses."""
+        self.ca_batch_query = """
+            epv = [];packages.each {g.V().has('pecosystem', ecosystem).has('pname', it.name)
+            .has('version', it.version).as('version', 'cve').select('version').in('has_version')
+            .dedup().as('package').select('package', 'version', 'cve')
+            .by(valueMap()).by(valueMap()).by(out('has_snyk_cve')
+            .valueMap().fold()).fill(epv);};epv;
```

Comment:
`ca_batch_query` need not be instance state, can be a class scoped variable or global

Reply:
Ack!